### ISSUE TITLE
Crash when change Effect Opacity value

### ISSUE THREAD

**Issue by mixsoupe** [mobile-bungalow/tweak_shader_ae_rs#13]
If we change the "Effect Opacity" value in the "Compositing Option" of Tweak Shader, After Effect crashes, no matter which shader is loaded (and even if there is no shader loaded).
![image](https://github.com/user-attachments/assets/5f1b278f-8e10-4195-9f44-3fdc7f14e532)

**Comment by mobile-bungalow** [comment]
apparently when the opacity is not 100% the BPC value can't be relied upon and is 0, regardless of the actual format of the underlying output image. what a hassle.

**Comment by mobile-bungalow** [comment]
[I removed the explicit panic](https://github.com/mobile-bungalow/tweak_shader_ae_plugin/commit/490dd66624d1a6d592ffb16b6563218c82ca0317#diff-ed12f5ea605f23bb4d26cc65778e3daff9db3eec40e2abfc82beafca130a99a0L105-R121). The most recent build should work. I will run a release soon. It seems that the BPC value is only 0 while adjusting the opacity knob. It should be fixed but I wouldn't be surprised if the behavior has some new issues. I've only tested it on MacOs. 

unrelated, I would absolutely accept some example shaders if you wanted to open a PR.
